src/guides/collator_deployment.md

```diff
@@ -0,0 +1,182 @@
+# Introduction
+Collators are members of the network that maintain the parachains they take part in. Collators produce the state transition proof for relay chain validators but also author the blocks for the parachain and pass them to relay chain. There are some requirements that need to be considered prior to becoming a collator candidate including machine, bonding, account, and community requirements. For more info regarding collators, reference [collator section](https://wiki.polkadot.network/docs/learn-collator) on Polkadot wiki page.
+
+This doc demonstrates how to add new collators to an already running parachain.
+
+# Hardware Requirements
+Make sure your instance has at least the minimum recommended harware requirements:
+- CPU: 2 Cores - Intel Xeon E-2386/2388 or Ryzen 9 5950x/5900x
+- Memory: 8GB 
+- Disk: Depending on the chain size - 200GB SSD/nvme
+
+# Launch the Collator 
+1- Generate an Account
+2- Generate the Node Keys
+3- Run the Collator
+4- Bind The Account With Collator's Session Keys
+5- Bond Funds
+
+
+## 1- Create an Account
+This account is used to hold the funds.
+
+```
+> subkey generate
+Secret phrase:       crane material doll wife text jaguar cross donor crane caught soft permit
+  Network ID:        substrate
+  Secret seed:       0x79613c5ccb1c8aa9c21e1ddb2fbf65c8e979649cd52cfe041391c9f9d07ec3e9
+  Public key (hex):  0x4a503b20b3ada9ff3bf985271b5756e650a83ae47e44a7552dc2415ce8060a52
+  Account ID:        0x4a503b20b3ada9ff3bf985271b5756e650a83ae47e44a7552dc2415ce8060a52
+  Public key (SS58): 5Dk9EY7iUbjLX6fTDnbFZVGdnN7v2NgQtxj4Y7YjjZXgpCjE
+  SS58 Address:      5Dk9EY7iUbjLX6fTDnbFZVGdnN7v2NgQtxj4Y7YjjZXgpCjE
+```
+
+Store the secret phrase in a secure place. This account will be used on PolkadotJS app to bind you collator session keys to it.
+
+## 2- Generate the Node Keys
```

Comment:
This is a good explanation but it should be added to a guide for bootnode deployment or the general parachain deployment guide. It is not mandatory for collators.